recipes/Oasis/meta.yaml

```diff
@@ -1,44 +0,0 @@
-{% set name = "Oasis" %}
```

Comment:
We recommend to not delete recipes b/c that may prevent a feedstock from getting created. Is this from another PR? If so please restore it.

Reply:
ok I restored it, now i have to keep both libraries in the recipes all the time ?